src/Templates/SearchResultsProductTemplate.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Templates;
+
+/**
+ * ProductSearchTemplate class.
+ *
+ * @internal
+ */
+class SearchResultsProductTemplate {
+
+	const SLUG        = 'search-results-product';
```

Comment:
If possible, I would change the slug as well:
```suggestion
	const SLUG        = 'product-search-results';
```